src/mypreferencesmenu.html

```diff
@@ -3,7 +3,7 @@
         <div class="readOnlyContent" style="margin: 0 auto;">
             <div class="verticalSection verticalSection-extrabottompadding">
                 <h2 class="sectionTitle headerUsername" style="padding-left:.25em;"></h2>
-                <a is="emby-linkbutton" data-ripple="false" href="#" class="lnkMyProfile listItem-border">
+                <a is="emby-linkbutton" data-ripple="false" href="#" style="display:block;padding:0;margin:0;" class="lnkMyProfile listItem-border">
```

Comment:
This was causing the weird button issues in the user preference menu. The two elements that were always working had inline styles, which isn't ideal but now at least everything is working.

Reply:
Is this the issue where sometimes the menu items are too large?

Reply:
This might fix that as well but it definitely fixes the vertical alignment, which might be what you're talking about.